src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -18,4 +18,8 @@ public int countOut() {
 
         return ret;
     }
+
+    public DefaultCountingOutRhymer getTemp() {
```

Comment:
Shouldn't be encapsulated (typical private field). Using private fields internally usually do not require encapsulation.